homeassistant/components/met/weather.py

```diff
@@ -103,22 +108,22 @@ def should_poll(self):
     def unique_id(self):
         """Return unique ID."""
         if self.track_home:
-            return "home"
+            return f"home{self._name_appendix}"
```

Comment:
ah so was it daily or hourly as default before?

Reply:
I believe the daily forecast exists today.  Also, I didn't read through this to find out, but you might want to consider disabling the new 3 hour entity by default (at least the one created during onboarding).  Otherwise there will be two weather cards shown during onboarding.  Maybe that is the intent though.

Reply:
ah i was wondering about maybe disabled by default but thought it was ok given shared data but now thinking about onboarding hum. weather platform is so much fun!

Reply:
it does seem that daily is default and the way this only appends hourly and not daily to the string seems ok (no breaking) if im followly the code properly

Reply:
You are right. This is okay then.  Still may want to consider disabling the new entity